website/docs/use_cases/multi_data_centers.md

```diff
@@ -21,7 +21,7 @@ Suppose our application will be deployed in three data centers in a production e
 and these data centers are still under construction. Now we want to test the impact of
 such a deployment topology on the business in advance.
 
-Here we use TiDB cluster as an example. Suppose we already install the [TiBD cluster](https://docs.pingcap.com/tidb-in-kubernetes/stable/) and [Chaos Mesh](get_started/installation.md)
+Here we use TiDB cluster as an example. Suppose we already install the [TiDB cluster](https://docs.pingcap.com/tidb-in-kubernetes/stable/) and [Chaos Mesh](../get_started/installation.md)
```

Comment:
Great! I wonder why these links are not really dead on chaos-mesh.org :man_facepalming:

Reply:
I'm not familiar with front-end, but I guess that is a kind of customized link format, which would work for the website. But since it's customized, it does not work for markdown itself